### PR TITLE
Update Trivia/plugin.py

### PR DESCRIPTION
**Pull request by inhahe**
add @next and @hint, and strip formatting from answers before checking

### CODE REVIEW

File: Trivia/plugin.py [hunk 1]
```diff
@@ -301,6 +320,30 @@ def start(self, irc, msg, args, channel, num):
         irc.noReply()
     start = wrap(start, ['channel', optional('positiveInt')])
 
+    @internationalizeDocstring
+    def hint(self, irc, msg, args, channel):
+        """
```
Comment (ProgVal) [inline Trivia/plugin.py:325]: add `[<channel>]` here

File: Trivia/plugin.py [hunk 2]
```diff
@@ -301,6 +320,30 @@ def start(self, irc, msg, args, channel, num):
...
+        channel = ircutils.toLower(channel)
+        if channel in self.games:
+            self.games[channel].hintcommand()
+        else:    
+            irc.reply("Trivia is currently not active in this channel.")
+    hint = wrap(hint, ['channel'])
+
+    @internationalizeDocstring
+    def next(self, irc, msg, args, channel):
+        """
```
Comment (ProgVal) [inline Trivia/plugin.py:337]: same

File: Trivia/plugin.py [hunk 3]
```diff
@@ -301,6 +320,30 @@ def start(self, irc, msg, args, channel, num):
...
+    @internationalizeDocstring
+    def hint(self, irc, msg, args, channel):
+        """
+        
+        Invokes the next hint for the current question"""
+        channel = ircutils.toLower(channel)
+        if channel in self.games:
+            self.games[channel].hintcommand()
+        else:    
+            irc.reply("Trivia is currently not active in this channel.")
```
Comment (ProgVal) [inline Trivia/plugin.py:332]: Could you wrap the string in `_()` like the other commands? (it makes them translatable)

File: Trivia/plugin.py [hunk 4]
```diff
@@ -301,6 +320,30 @@ def start(self, irc, msg, args, channel, num):
...
+    @internationalizeDocstring
+    def next(self, irc, msg, args, channel):
+        """
+        
+        Moves onto the next question."""
+        channel = ircutils.toLower(channel)
+        if channel in self.games:
+            self.games[channel].skip()
+        else:    
+            irc.reply("Trivia is currently not active in this channel.")
```
Comment (ProgVal) [inline Trivia/plugin.py:344]: same

File: Trivia/plugin.py [hunk 5]
```diff
@@ -214,11 +214,30 @@ def event():
...
+                    schedule.removeEvent('next_%s' % self.channel)
+                except KeyError:
+                    pass
+                self.hint()
+                
         def answer(self, msg):
             correct = False
             for ans in self.a:
-                dist = self.DL(str.lower(msg.args[1]), str.lower(ans))
+                dist = self.DL(ircutils.stripFormatting(str.lower(msg.args[1])), str.lower(ans))
```
Comment (ProgVal) [inline Trivia/plugin.py:240]: Why this change?

**Comment by inhahe** [comment]
Just in case someone tries to answer who runs a script that colors all of
their messages or does some other formatting to it. I mean.. I guess hardly
everyone does that, but I do (in certain channels), and it was getting in
the way of my answering trivia questions. =P

**Comment by inhahe** [comment]
s/hardly everyone/hardly anyone/

**Comment by ProgVal** [comment]
ah, fair

File: Trivia/plugin.py [hunk 6]
```diff
@@ -214,11 +214,30 @@ def event():
             if self.active:
                 schedule.addEvent(event, eventTime, 'next_%s' % self.channel)
 
-
+        def skip(self):
+          self.reply("Question skipped. The answer was: %s" % self.a[0])
```
Comment (ProgVal) [inline Trivia/plugin.py:218]: Missing `_()` here too. Sorry I should have spotted it earlier.

```
self.reply(_("Question skipped. The answer was: %s") % self.a[0])
```

File: Trivia/plugin.py [hunk 7]
```diff
@@ -214,11 +214,30 @@ def event():
             if self.active:
                 schedule.addEvent(event, eventTime, 'next_%s' % self.channel)
 
-
+        def skip(self):
+          self.reply(_("Question skipped. The answer was: %s" % self.a[0]))
```
Comment (ProgVal) [inline Trivia/plugin.py:218]: No, it should be: `self.reply(_("Question skipped. The answer was: %s") % self.a[0])`.

The reason is that _() returns a translation of the template instead of the final string. (Templates can be translated ahead of time)
